src/applications/components/ApplicationItem.vue

```diff
@@ -42,7 +42,7 @@
           >
             <RouterLink
               place="userName"
-              @click.native.stop=""
+              @click.native.stop
```

Comment:
Need to check if that still does the job. I think the goal was to _not_ open the application when clicking on the person who decided it.